tests/test_job.py

```diff
@@ -158,6 +161,7 @@ def test_submit_job_local_file_with_success(self, mocker, mock_session, make_moc
             'speaker_channels_count': 1,
             'filter_profanity': True,
             'remove_disfluencies': True,
+            'language' = LANG,
             'delete_after_seconds': 0
         }
         response = make_mock_response(url=JOB_ID_URL, json_data=data)
```

Comment:
You will need to modify a few lines lower to pass LANG to `client.submit_job_local_file(FILENAME, METADATA, CALLBACK_URL, True,True, 1, CUSTOM_VOCAB, True, True, 0, LANGUAGE)`